tests/capture/test_templates.py

```diff
@@ -257,6 +257,7 @@ def fn(*args):
     qml.MPSPrep,
     qml.GQSP,
     qml.QROMStatePreparation,
+    qml.SelectPauliRot,
```

Comment:
Does the default test fail? if so, why?

Reply:
With new templates I always have to create the particular test 🤔 
If that is not the case, happy to know about it 🙌